Makefile.am

```diff
@@ -522,6 +522,28 @@ endif #DOXYMAN
 
 endif #FAPI
 
+### Helper Functions ###
+define make_parent_dir
+    if [ ! -d $(dir $1) ]; then mkdir -p $(dir $1); fi
+endef
+
+define make_tss_user_and_group
+    groupadd -r tss && \
```

Comment:
I think we should change this to not fail if the user already exists:
```
    (id -g tss || groupadd -r tss) && \
    (id -u tss || useradd -r -g tss tss)
```

Reply:
I totally agree.